crates/autopilot/src/solvable_orders.rs

```diff
@@ -239,14 +235,6 @@ impl SolvableOrdersCache {
 
         self.database.store_order_events(&order_events).await;
 
-        let id = if self.store_in_db {
-            let id = self.database.replace_current_auction(&auction).await?;
-            tracing::info!(auction = %id, %block, "stored new auction in database");
```

Comment:
I think we should still write the auction to the DB. Otherwise the `/auction` endpoint would break can be a useful tool for debugging.

Reply:
We do it here for the colocated setup https://github.com/cowprotocol/services/blob/main/crates/autopilot/src/run_loop.rs#L93